README.md

```diff
@@ -12,11 +12,27 @@ npm install --save-dev broccoli-merge-trees
 ```
 
 ## Usage
+1. With `new`
+```js
+const BroccoliMergeTrees = require('broccoli-merge-trees');
 
+let mergedNode = new BroccoliMergeTrees(inputNodes, options);
+```
```

Comment:
This seems wrong, we shouldn't suggest using `new` with the plain function form. Cases 1 and 3 in this list should be collapsed to a single entry.

```js
const { MergeTrees } = require('broccoli-merge-trees');

let mergedNode = new MergeTrees(inputNodes, options);
```